client/dist/styles/ratingsReviews.css

```diff
@@ -639,4 +646,35 @@
   outline: none;
   border: 1px solid darkgray;
   font-family: arial;
+}
+
+.r-photo-wrapper {
+  padding: 15px;
+}
+
+.r-photos {
+  padding-top: 10px;
+  display: flex;
+}
+
+.r-photo-title {
+  padding-right: 5px;
+}
+
+.r-photo-button {
+  height: 25px;
+  padding: 0px 5px 0px 5px;
+  background-color: white;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.r-photo-button:hover,
+.r-submit:hover {
+  background-color: rgb(235, 235, 235);
+}
+
+.r-required {
+  color: red;
+  font-size: small;
 }
```

Comment:
just being nit picky but both yours and my CSS file have a grip of PX values.  i think leslie was saying to not use PX, so a good refactor could possibly be to use rem or em?  not immediate or anything just suggestion